project_liberation/requirements.txt

```diff
@@ -0,0 +1,9 @@
+black
+Django
```

Comment:
Is the project really prepared to run with any version of Django? No constraints at all?

Reply:
For this moment I don't want to add any constraints. If any future update will make some problems then I will add constraint to version.